src/types/staff.ts

```diff
@@ -2,8 +2,14 @@ export interface StaffResult {
   firstName: string;
   lastName: string;
   emailAddress: string;
-  patchId: string | null;
-  areaId: string | null;
+  patches: Patches[];
+}
+
+export interface Patches {
```

Comment:
lets not use plural here. I recommend PatchResult or Patch

Reply:
I have updated the class